tests/controller_tests/test_ingest.py

```diff
@@ -592,21 +596,37 @@ def test_save_submission_file(test_client):
     assert Submission.query.first().submission_file == filebytes
 
 
-def test_next_submission_numpy_type(test_client):
-    """
-    Postgres cannot parse numpy ints. Test we cast them correctly.
-
-    NB, this test not appropriate if app used with SQLlite, as that can parse numpy types. Intended for PostgreSQL.
-    """
-    sub = Submission(
-        submission_id="S-R01-3",
-        reporting_period_start=datetime.now(),
-        reporting_period_end=datetime.now(),
-        reporting_round=1,
-    )
-    db.session.add(sub)
-    sub_id = next_submission_id(reporting_round=np.int64(1))
-    assert sub_id == "S-R01-4"
+@pytest.mark.parametrize(
```

Comment:
parameterisation might be overkill for this one, only need to test that numpy type is handled the once?

Reply:
Yeh, you're right there. Rewritten as one test.